From cbbb384084f341875314cf4c8bb8bc92a7dcad53 Mon Sep 17 00:00:00 2001
From: Mathias Fiedler <mathias_fiedler@mentor.com>
Date: Tue, 28 Jul 2015 03:55:07 +0200
Subject: [PATCH 11/26] ionbuf: rename meta functions to be exported

Added a gst_ prefix so they are exported with gstreamer's export regex
pattern.
---
 gst-libs/gst/ionbuf/gstionbuf_meta.c | 4 ++--
 gst-libs/gst/ionbuf/gstionbuf_meta.h | 8 ++++----
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/gst-libs/gst/ionbuf/gstionbuf_meta.c b/gst-libs/gst/ionbuf/gstionbuf_meta.c
index f675e8c..6043feb 100644
--- a/gst-libs/gst/ionbuf/gstionbuf_meta.c
+++ b/gst-libs/gst/ionbuf/gstionbuf_meta.c
@@ -1,7 +1,7 @@
 #include "gstionbuf_meta.h"
 
 GType
-ionBufFd_meta_api_get_type (void)
+gst_ionBufFd_meta_api_get_type (void)
 {
   static volatile GType type;
   static const gchar *tags[] = { NULL };
@@ -41,7 +41,7 @@ ionBufFd_meta_transform (GstBuffer * transbuf, GstMeta * meta,
 }
 
 const GstMetaInfo *
-ionBufFd_meta_get_info (void)
+gst_ionBufFd_meta_get_info (void)
 {
   static const GstMetaInfo *meta_info = NULL;
 
diff --git a/gst-libs/gst/ionbuf/gstionbuf_meta.h b/gst-libs/gst/ionbuf/gstionbuf_meta.h
index 84c42ff..37ffba2 100644
--- a/gst-libs/gst/ionbuf/gstionbuf_meta.h
+++ b/gst-libs/gst/ionbuf/gstionbuf_meta.h
@@ -21,13 +21,13 @@ struct _GstIonBufFdMeta {
 
 typedef struct _GstIonBufFdMeta GstIonBufFdMeta;
 
-GType ionBufFd_meta_api_get_type (void);
-const GstMetaInfo * ionBufFd_meta_get_info (void);
+GType gst_ionBufFd_meta_api_get_type (void);
+const GstMetaInfo * gst_ionBufFd_meta_get_info (void);
 
-#define ION_BUF_FD_META_API_TYPE (ionBufFd_meta_api_get_type())
+#define ION_BUF_FD_META_API_TYPE (gst_ionBufFd_meta_api_get_type())
 #define gst_buffer_get_ionfd_meta(b)		\
   ((GstIonBufFdMeta*)gst_buffer_get_meta((b),ION_BUF_FD_META_API_TYPE))
-#define ION_BUF_FD_META_INFO (ionBufFd_meta_get_info ())
+#define ION_BUF_FD_META_INFO (gst_ionBufFd_meta_get_info ())
 
 GstIonBufFdMeta * gst_buffer_add_ionBufFd_meta (GstBuffer * buffer,
 					     unsigned int fd, unsigned int offset, unsigned int size);
-- 
1.9.1

